site/src/components/FeaturePanels.js

```diff
@@ -1,23 +1,22 @@
 import React from 'react';
-import featurePanelStyles from '@site/src/components/css/FeaturePanel.module.css';
 
 function FeaturePanel({Svg, title, text, link}) {
     return (
-        <a className={featurePanelStyles.panel} href={link}>
-            <div className={featurePanelStyles.header}>
-                <Svg className={featurePanelStyles.icon}/>
-                <div className={featurePanelStyles.title}>{title}</div>
+        <a className="panel" href={link}>
+            <div className="header">
```

Comment:
This is really generic. Would it make sense to prepend these with feature? Like `feature-header` `feature-icon` `feature-title` or maybe `panel-header` `panel-icon` `panel-title`?